usage/html.go

```diff
@@ -36,46 +36,57 @@ func markdownHelpAction(ctx *cli.Context) error {
 	if err != nil {
 		return errs.FileError(err, index)
 	}
-	markdownHelpPrinter(w, mdAppHelpTemplate, ctx.App)
+	markdownHelpPrinter(w, mdAppHelpTemplate, "", ctx.App)
 	if err := w.Close(); err != nil {
 		return errs.FileError(err, index)
 	}
 
+	// preserve jekyll compatibility for transition period
+	fileName := "index.md"
+	if ctx.IsSet("hugo") {
```

Comment:
This should be `ctx.Bool("hugo")`, because `--hugo=false` will set hugo, but it will be false.

Reply:
done